custom/aaa/dbaccessors.py

```diff
@@ -0,0 +1,306 @@
+from __future__ import absolute_import, unicode_literals
+
+from dateutil.relativedelta import relativedelta
+
+from custom.aaa.models import Child, ChildHistory
+
+
+class ChildQueryHelper(object):
+    def __init__(self, domain, person_case_id):
+        self.domain = domain
+        self.person_case_id = person_case_id
+
+    def infant_details(self):
+        extra_select = {
+            'breastfeedingInitiated': 'breastfed_within_first',
+            'dietDiversity': 'diet_diversity',
+            'birthWeight': 'birth_weight',
+            'dietQuantity': 'diet_quantity',
+            'breastFeeding': 'comp_feeding',
+            'handwash': 'hand_wash',
+            'exclusivelyBreastfed': 'is_exclusive_breastfeeding',
+            'babyCried': 'child_cried',
+            'pregnancyLength': "'N/A'",
+        }
+
+        return Child.objects.extra(
+            select=extra_select
+        ).values(*list(extra_select.keys())).get(
+            domain=self.domain,
+            person_case_id=self.person_case_id,
+        )
+
+    def postanatal_care_details(self):
+        # TODO update when CcsRecord will have properties from PNC form
+        return [{
+            'pncDate': 'N/A',
+            'breastfeeding': 'N/A',
+            'skinToSkinContact': 'N/A',
+            'wrappedUpAdequately': 'N/A',
+            'awakeActive': 'N/A',
+        }]
+
+    def vaccination_details(self, period):
+        if period == 'atBirth':
+            return dict(
+                vitamins=[
```

Comment:
curious why these are called vitamins?

Reply:
i'm not sure. Some of these are technically vitamins. I'm not going to change them here because the UI is currently dependent on these names. I think large scale renames would add too much noise to this PR